client/dashboard/index.js

```diff
@@ -25,7 +25,7 @@ export default class Dashboard extends Component {
 				<StorePerformance />
 				<div className="woocommerce-dashboard__columns">
 					<div>
-						<TopSellingProducts />
+						<TopSellingProducts query={ query } path={ path } />
```

Comment:
I'm not seeing `path` as a prop type in top selling products or the leader board component.  Is this used somewhere else?